docs/using.rst

```diff
@@ -10,12 +10,12 @@ User Guide
 Installation
 ============
 
-.. _letsencrypt-auto:
+.. _certbot-auto:
```

Comment:
Similarly, `certbot-auto` doesn't exist yet. We can either wait to merge this or revert these changes for the time being.